adyenv6core/src/com/adyen/v6/factory/AdyenRequestFactory.java

```diff
@@ -321,15 +327,13 @@ public CaptureRequest createCaptureRequest(final String merchantAccount, final B
                                                      .merchantAccount(merchantAccount)
                                                      .originalReference(authReference)
                                                      .reference(merchantReference);
-        ApplicationInfo applicationInfo = updateApplicationInfo(request.getApplicationInfo());
-        request.setApplicationInfo(applicationInfo);
+        updateApplicationInfo(request.getApplicationInfo());
```

Comment:
where is the updateApplicationInfo defined is it already in the code base ?

Reply:
yes it is.

Reply:
great!